truffle-config.js

```diff
@@ -68,6 +47,51 @@ const manualNetworks = {
   },
 }
 
+/**
+ * You shouldn't change anything below this line to preserve this Truffle Box's automatic handling of target realms,
+ * chains, networks, compiler versions, etc.
+ *
+ * If you want to configure anything related to networks, please customize the `manualNetworks` object above.
+ *
+ * Target realms (ethereum / boba / celo / conflux / etc.) are read from multiple possible sources, in this order:
+ *  1. `--network <realm>.<chain>` argument
+ *  2. `WITNET_EVM_REALM` environment variable
+ *  3. default (ethereum)
+ */
+const witnetSettings = require("./node_modules/witnet-ethereum-bridge/migrations/witnet.settings")
+let realm = process.env.WITNET_EVM_REALM ? process.env.WITNET_EVM_REALM.toLowerCase() : "default"
+const args = process.argv.join("=").split("=")
+const networkIndex = args.indexOf("--network")
+if (networkIndex >= 0) {
+  realm = (args[networkIndex + 1] || "default").split(".")[0]
+}
+console.info(`Configuring "${realm}" target realm...
+==============================${"=".repeat(realm.length)}
+> Default realm is "ethereum"
+> Target realms and chains can be specified with "--network <realm>.<chain>"
+> Available realms and chains can be found in "migrations/2_user_contracts.js"`)
+
+// Use realm-specific compiler settings
+const compilers = { ...witnetSettings.compilers.default, ...witnetSettings.compilers[realm] }
+
+const supportedNetworks = Object.entries(witnetSettings.networks).reduce((acc, [realmKey, realmVal]) => {
```

Comment:
Comment describing what's being done would be helpful: is it something like "making sure all configured networks are prefixed with a valid realm" ?

Reply:
Done! Thanks!